support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/ticket/registry/CassandraTicketRegistryCleaner.java

```diff
@@ -0,0 +1,35 @@
+package org.apereo.cas.ticket.registry;
+
+import org.apereo.cas.logout.LogoutManager;
+import org.apereo.cas.ticket.TicketGrantingTicket;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.scheduling.annotation.Scheduled;
+
+/**
+ * @author David Rodriguez
+ *
+ * @since 5.2.0
+ */
+public class CassandraTicketRegistryCleaner implements TicketRegistryCleaner {
+
+    private final CassandraTicketRegistryDao ticketRegistryDao;
+    private final LogoutManager logoutManager;
+
+    public CassandraTicketRegistryCleaner(@Qualifier("cassandraTicketRegistry") final CassandraTicketRegistryDao ticketRegistry,
+                                          final LogoutManager logoutManager) {
+        this.ticketRegistryDao = ticketRegistry;
+        this.logoutManager = logoutManager;
+    }
+
+    @Scheduled(initialDelayString = "${cas.ticket.registry.cleaner.startDelay:20000}",
+            fixedDelayString = "${cas.ticket.registry.cleaner.repeatInterval:60000}")
+    @Override
+    public void clean() {
+        ticketRegistryDao.getExpiredTgts().forEach(ticket -> {
```

Comment:
This does not look right to me. Why are we only cleaning expired TGTs? The cleaner should not make any assumptions about ticket types it works with.